### ISSUE TITLE
Small Accessibility Tweaks

### ISSUE THREAD

**Issue by shortwavesurfer2009** [dessalines/jerboa#616]
** Jerboa Version ** 0.0.34

**Describe the bug**
Some small accessibility tweaks are still required. Screenshots and suggestions below.

- [ ] in screenshot 1 talkback reads "Todo DeGoogle Yourself at lemmy ml". It should not read " todo" which i am assuming is likely coming from the generic forum icon to the left as other subs that have icons do not do this. 
![Jerboa 1](https://github.com/dessalines/jerboa/assets/116814522/46e05577-99ad-4845-a520-38b6ca419241)

- [ ] in screenshot 2 the "select listing filter" button is highlighted. To make it clearer i suggest changing the string to "select feed" 
![Jerboa 2_1](https://github.com/dessalines/jerboa/assets/116814522/bd353646-02a7-4614-9036-054efb7af05f)

- [ ] in screenshot 3 the "select sort option" button is highlighted. I suggest replacing the string with "sort" or "sort by" 
![Jerboa 3_1](https://github.com/dessalines/jerboa/assets/116814522/9353647e-7016-45a2-a7f1-e405f2c9525e)

- [ ] in screenshot 4 the "show more options" button is highlighted. I suggest changing the string to "more options" 
![Jerboa 4_1](https://github.com/dessalines/jerboa/assets/116814522/e0cf8301-b706-4149-828f-787c2fe953c1)

- [ ] in screenshot 5 the "toggle vote" button is highlighted for upvoting a post (and i assume comments use it too and this applies. This button should read as "upvote" when in this state and "upvoted" when it has been pressed. 
![Jerboa 5_1](https://github.com/dessalines/jerboa/assets/116814522/980f9d8f-ee00-4da9-a7a3-44b47f1a3081)

- [ ] in screenshot 6 the "toggle vote" button is highlighted for downvoting a post (and i assume comments use it too and this applies. This button should read as "downvote" when in this state and "downvoted" when it has been pressed. 
![Jerboa 6_1](https://github.com/dessalines/jerboa/assets/116814522/961a747c-06e2-4506-aac2-d9bbde70b640)

Edit 1:
- [ ] the "select inbox filter" should change to "filter"
- [ ] the "select sort option" in the bookmarks tab should be changed to "sort" or "sort by". See screenshot 7. 
![Screenshot_20230613-173008_Jerboa_for_Lemmy](https://github.com/dessalines/jerboa/assets/116814522/6bcbfd6e-5c51-4819-9630-bd052e7b73d8)
- [ ] on the profile page the " select sort option" button should be changed to "sort" or "sort by". 
![Screenshot_20230613-173247_Jerboa_for_Lemmy](https://github.com/dessalines/jerboa/assets/116814522/c0f65763-e514-4171-9e58-f680b84cd821)
